examples/12_custom_secrets.py

```diff
@@ -24,9 +21,11 @@
 )
 
 # Tools
+register_tool("BashTool", BashTool)
+register_tool("FileEditorTool", FileEditorTool)
 tools = [
-    BashTool.create(working_dir=os.getcwd()),
-    FileEditorTool.create(),
+    ToolSpec(name="BashTool", params={"working_dir": os.getcwd()}),
+    ToolSpec(name="FileEditorTool", params={}),
```

Comment:
```suggestion
    ToolSpec(name="FileEditorTool"),
```

Is it possible to just provide the tool name and not have to pass the empty param in this case?